entrypoint.sh

```diff
@@ -83,18 +83,31 @@ fi
 # We assume the the patch/Z version is at least 11, based on the current edit
 # Thus it will definitely have both the RBAC and create_user features
 AIRFLOW_VERSION="$(airflow version)"
+AIRFLOW_X_VERSION="$(echo ${AIRFLOW_VERSION} | cut -d . -f 1)"
 AIRFLOW_Y_VERSION="$(echo ${AIRFLOW_VERSION} | cut -d . -f 2)"
 
+echo "${AIRFLOW_VERSION}, ${AIRFLOW_X_VERSION}, ${AIRFLOW_Y_VERSION}"
+
 # Requires 'rbac' mode to be set to true to run the command properly
-if check_set "${ENABLE_AIRFLOW_RBAC_SETUP_AUTH}" && [ "${AIRFLOW_Y_VERSION}" -eq 10 ]; then
+if check_set "${ENABLE_AIRFLOW_RBAC_SETUP_AUTH}"; then
   echo "Adding user for Airflow Web UI RBAC login..."
-  gosu "${AIRFLOW_USER}" airflow create_user \
-    -r "${AIRFLOW_WEBSERVER_RBAC_ROLE}" \
-    -u "${AIRFLOW_WEBSERVER_RBAC_USER}" \
-    -p "${AIRFLOW_WEBSERVER_RBAC_PASSWORD}" \
-    -e "${AIRFLOW_WEBSERVER_RBAC_EMAIL}" \
-    -f "${AIRFLOW_WEBSERVER_RBAC_FIRST_NAME}" \
-    -l "${AIRFLOW_WEBSERVER_RBAC_LAST_NAME}"
+  if [ "${AIRFLOW_X_VERSION}" == "1" ] && [ "${AIRFLOW_Y_VERSION}" -ge "10" ]; then
```

Comment:
Since you are using `-ge`, for older shell support consistency, let's switch `==` to `eq`?

Ref:
- https://unix.stackexchange.com/a/16110
- https://tldp.org/LDP/abs/html/comparison-ops.html